scripts/sct_label_vertebrae.py

```diff
@@ -849,7 +849,7 @@ def label_segmentation(fname_seg, list_disc_z, list_disc_value, fig_anat_straigh
     """
 
     if fig_anat_straight is None:
-        fig_anat_straight = Param.fig_anat_straight
+        fig_anat_straight = Param().fig_anat_straight
```

Comment:
I see what is happening, it is confusing to have both `param` and `Param` used throughout the file